packages/sdk/src/streamStateView_Members_Membership.ts

```diff
@@ -40,6 +40,16 @@ export class StreamStateView_Members_Membership {
         }
     }
 
+    info(userId: string): MembershipOp {
+        const isJoined = this.joinedUsers.has(userId)
+        if (isJoined) return MembershipOp.SO_JOIN
+        const isInvited = this.invitedUsers.has(userId)
+        if (isInvited) return MembershipOp.SO_INVITE
+        const isLeft = !isJoined && !isInvited
+        if (isLeft) return MembershipOp.SO_LEAVE
```

Comment:
can you use `readonly leftUsers = new Set<string>()` to return LEAVE? And handle UNSPECIFIED in the ui as well?

Reply:
> handle UNSPECIFIED in the ui as well?

What do you mean?

Reply:
Sorry, not in this pr, I was just calling out that UNSPECIFIED is a valid state that is different than LEAVE. In our system if you have a LEAVE it means you recently left and we can put up a time and an event hash for that action. UNSPECIFIED means they either never joined or they left before our last snapshot